src/app/util/attribute-storage.cpp

```diff
@@ -893,19 +893,19 @@ static uint8_t findClusterEndpointIndex(EndpointId endpoint, ClusterId clusterId
         {
             break;
         }
-        epi = static_cast<uint8_t>(epi +
-                                   ((emberAfFindClusterIncludingDisabledEndpointsWithMfgCode(emAfEndpoints[i].endpoint, clusterId,
-                                                                                             mask, manufacturerCode) != NULL)
-                                        ? 1
-                                        : 0));
+        epi = static_cast<uint16_t>(epi +
+                                    ((emberAfFindClusterIncludingDisabledEndpointsWithMfgCode(emAfEndpoints[i].endpoint, clusterId,
+                                                                                              mask, manufacturerCode) != NULL)
+                                         ? 1
+                                         : 0));
     }
 
     return epi;
 }
 
-static uint8_t findIndexFromEndpoint(EndpointId endpoint, bool ignoreDisabledEndpoints)
+static uint16_t findIndexFromEndpoint(EndpointId endpoint, bool ignoreDisabledEndpoints)
```

Comment:
At some point, maybe we should have an EndpointIndex type?   Followup, presumably.